sdk/formrecognizer/Azure.AI.FormRecognizer/MigrationGuide.md

```diff
@@ -0,0 +1,737 @@
+
+Familiarity with `Azure.AI.FormRecognizer (3.1.x and below)` package is assumed. For those new to the Azure Form Recognizer client library for .NET please refer to the [README][readme] rather than this guide.
+
+## Table of Contents
+- [Migration benefits](#migration-benefits)
+- [Important changes](#important-changes)
+    - [Client usage](#client-usage)
+    - [Analyzing document](#analyzing-documents)
+    - [Analyzing a document with a custom model](#analyzing-a-document-with-a-custom-model)
+    - [Training a custom model](#training-a-custom-model)
+    - [Manage models](#manage-models)
+- [Additional samples](#additional-samples)
+
+## Migration benefits
+
+A natural question to ask when considering whether to adopt a new version of the library is what the benefits of doing so would be. As Azure Form Recognizer has matured and been embraced by a more diverse group of developers, we have been focused on learning the patterns and practices to best support developer productivity and add value to our customers.
+
+There are many benefits to using the new design of the `Azure.AI.FormRecognizer (4.0.x)` library. This new version of the library introduces two new clients `DocumentAnalysisClient` and the `DocumentModelAdministrationClient` with unified methods for analyzing documents and provides support for the new features added by the service in API version `2021-09-30-preview` and later.
+
+New features provided by the `DocumentAnalysisClient` include:
+- One consolidated method for analyzing document layout, a general prebuilt document model type, along with the same prebuilt models that were included previously (receipts, invoices, business cards, identity documents), and custom models.
+- Models introduced in the latest version of the library, such as `AnalyzeResult`, remove hierarchical dependencies between document elements and move them to a more top level and easily accessible position.
+- The Form Recognizer service has further improved how to define where elements are located on documents by moving towards `BoundingRegion` definitions allowing for cross-page elements.
+- Document element fields are returned with more information, such as content and spans. 
+
+New features provided by the `DocumentModelAdministrationClient` include:
+- Users can now assign their own model IDs and specify a description when building, composing, or copying models.
+- Listing models now includes both prebuilt and custom models.
+- When using `GetModel()`, users can get the field schema (field names and types that the model can extract) for the model they specified, including for prebuilt models. 
+- Ability to get information from model operations that occurred in the last 24 hours.
+
+The below table describes the relationship of each client and its supported API version(s):
```

Comment:
nit: the table below?

Reply:
resolved in https://github.com/Azure/azure-sdk-for-net/pull/24398